drivers/media/platform/bcm2835/bcm2835-camera.c

```diff
@@ -54,6 +54,10 @@ int bcm2835_v4l2_debug;
 module_param_named(debug, bcm2835_v4l2_debug, int, 0644);
 MODULE_PARM_DESC(bcm2835_v4l2_debug, "Debug level 0-2");
 
+int video_nr = -1;
```

Comment:
Perhaps same argument should be applied to bcm2835_v4l2_debug above.  As you've seen, I've copy/pasted from somewhere else in the next patch and it has become static :-)